src/DacpacTool/Program.cs

```diff
@@ -33,6 +33,16 @@ static async Task<int> Main(string[] args)
             };
             buildCommand.Handler = CommandHandler.Create<BuildOptions>(BuildDacpac);
 
+            var inspectCommand = new Command("inspect")
```

Comment:
New invocation of `DacpacTool` to get a list of includes for checking incremental builds. I'm not sure if anyone has thoughts on what the command should be called, I don't love "inspect"... maybe "inspect-includes" or "get-includes"?

Reply:
Yeah, I was thinking of having a pre-process command that wouldn't just list the included files but would also generate the entire script to the `obj` folder so that we can then use that to create the final package. But then you wouldn't have the incremental build support. But maybe `collect-includes` would be an appropriate name?

Reply:
Renamed to `collect-includes`.